main.cpp

```diff
@@ -1,2 +1,6 @@
 int main() {
+  int a, b;
+  if (a != b)
```

Comment:
warning: statement should be inside braces [readability-braces-around-statements]

```suggestion
  if (a != b) {
```

main.cpp:4:
```diff
-     a = b;
+     a = b;
+ }
```